src/plugins/console/public/application/models/legacy_core_editor/legacy_core_editor.ts

```diff
@@ -40,6 +38,9 @@ export class LegacyCoreEditor implements CoreEditor {
     this.editor.setShowPrintMargin(false);
 
     const session = this.editor.getSession();
+    // @ts-ignore
```

Comment:
Would you mind using `@ts-expect-error` instead of `@ts-ignore` just so we know when this is fixed?